backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -234,6 +235,15 @@ public void enableGLDebugOutput (boolean enable, PrintStream debugOutputStream)
 		debugStream = debugOutputStream;
 	}
 
+	/** Whether to use the "glfw_async" library.
+	 *
+	 * This means you do not have to set the JVM argument "-XstartOnFirstThread"
+	 *
+	 * @see <a href= "https://libgdx.com/news/2021/07/devlog-7-lwjgl3#do-i-need-to-do-anything-else"> Documentation</a> */
+	public static void useGlfwAsync () {
+		Configuration.GLFW_LIBRARY_NAME.set("glfw_async");
```

Comment:
I gave this PR a test run and calling `Lwjgl3ApplicationConfiguration.useGlfwAsync()` causes a crash at startup on Windows 11. Since this only is helpful when called on MacOS (and is harmful everywhere else), shouldn't it have a check so it only runs on that OS? Like:
```java
if (SharedLibraryLoader.os == Os.MacOsX)
  Configuration.GLFW_LIBRARY_NAME.set("glfw_async");
```